plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergTrinoRestCatalogConnectorSmokeTest.java

```diff
@@ -186,14 +186,6 @@ public void testDropTableWithMissingManifestListFile()
                 .hasMessageContaining("Table location should not exist");
     }
 
-    @Test
-    @Override
-    public void testDropTableWithMissingDataFile()
```

Comment:
This test was failing expectedly after the upgrade because we made the change to remove Puffin files as part of https://github.com/apache/iceberg/pull/9305/files . We used to expect this test to fail in this override because Puffin files would linger after the "DROP" and we have an assertion that validates that all files are removed so the test would fail. Now, this override is no longer needed, we expect the base testDropTableWithMissingDataFile to pass. So we can just remove the test override and rely on the base.